python/paddle/fluid/initializer.py

```diff
@@ -569,6 +569,51 @@ def __call__(self, var, block):
         return op
 
 
+class RemoteInitializer(Initializer):
+    """Implements the remote initializer
+
+    Args:
+        value (float): constant value to initialize the variable
+
+    Examples:
+        .. code-block:: python
+
+            fc = fluid.layers.fc(input=x, size=10,
+                param_attr=fluid.initializer.Constant(value=2.0))
+    """
+
+    def __init__(self, eps=[]):
+        assert eps is not None
+        super(RemoteInitializer, self).__init__()
+        self._endpoints = eps
+        self._op_role_attr_name = op_proto_and_checker_maker.kOpRoleAttrName()
+        self._op_role_attr_value = op_proto_and_checker_maker.OpRole.RPC
+
+    def __call__(self, var, block):
+        """Add constant initialization ops for a variable
+
+        Args:
+            var: Variable that needs to be initialized
+            block: The block in which initialization ops
+                   should be added
+
+        Returns:
+            the initialization op
+        """
+        assert isinstance(var, framework.Variable)
+        assert isinstance(block, framework.Block)
+        op = block.append_op(
+            type="recv",
+            inputs={},
```

Comment:
not related to this PR: why `recv` has no input? Should it be the same as its output?